### ISSUE TITLE
manual/javaGuide/main/templates/JavaTemplates.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#968]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/templates/JavaTemplates.md
- diffs between [2.2.0/manual/javaGuide/main/templates/JavaTemplates.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/javaGuide/main/templates/JavaTemplates.md) and [2.3.5/manual/javaGuide/main/templates/JavaTemplates.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/templates/JavaTemplates.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/main/templates/JavaTemplates.md   2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/javaGuide/main/templates/JavaTemplates.md  2014-07-28 12:26:38.000000000 +0900
@@ -1,8 +1,9 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # The template engine

 ## A type safe template engine based on Scala

-Play comes with a powerful Scala-based template engine, whose design was inspired by ASP.NET Razor. Specifically it is:
+Play comes with <a href="https://github.com/playframework/twirl">Twirl</a>, a powerful Scala-based template engine, whose design was inspired by ASP.NET Razor. Specifically it is:

 - **compact, expressive, and fluid**: it minimizes the number of characters and keystrokes required in a file, and enables a fast, fluid coding workflow. Unlike most template syntaxes, you do not need to interrupt your coding to explicitly denote server blocks within your HTML. The parser is smart enough to infer this from your code. This enables a really compact and expressive syntax which is clean, fast and fun to type.
 - **easy to learn**: it allows you to quickly become productive, with a minimum of concepts. You use simple Scala constructs and all your existing HTML skills.
@@ -154,7 +155,7 @@
 <h1>@title("hello world")</h1>
 '''

-> **Note:** Declaring code block this way in a template can be sometime useful but keep in mind that a template is not the best place to write complex logic. It is often better to externalize these kind of code in a Java class (that you can store under the `views/` package as well if your want).
+> **Note:** Declaring code block this way in a template can be sometime useful but keep in mind that a template is not the best place to write complex logic. It is often better to externalize these kind of code in a Java class (that you can store under the `views/` package as well if you want).

 By convention a reusable block defined with a name starting with **implicit** will be marked as `implicit`:

@@ -193,7 +194,7 @@
 If you have common imports, which you need in all templates, you can declare in `build.sbt`

 '''scala
-templatesImport += "com.abc.backend._"
+TwirlKeys.templateImports += "org.abc.backend._"
 '''

 ## Comments
@@ -231,4 +232,4 @@
 </p>
 '''

-> **Next:** [[Common use cases | JavaTemplateUseCases]]
\ No newline at end of file
+> **Next:** [[Common use cases | JavaTemplateUseCases]]

```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1120
